src/lib/parsers/Maybe.ts

```diff
@@ -0,0 +1,96 @@
+import type { option } from 'lexure';
+
+/**
+ * A type used to express a value that may or may not exist.
+ * @typeparam T The value's type.
+ */
+export type Maybe<T> = Some<T> | None;
```

Comment:
Isn't maybe.. like Rust's Option? If it is...would that be a better name?

Reply:
I see `Maybe` and think of Haskell's `Maybe` (`Just a | Nothing`), but yeah, this is pretty much Rust's `Option` with `Some<T> | None` (and that's how lexure does it too). Not too arsed about the name; `Maybe` might be better in this case since the `Args` class has method for options already, and `nextOption` would be ambiguous.

Reply:
Then we might as well call `next` `some`, no? 👀

Reply:
That doesn't really make much sense to me.

Reply:
@vladfrangu some implies more than one, next is just the next one. I gotta agree with @Alcremie on that one.